build.rs

```diff
@@ -7,4 +7,16 @@ fn main() {
             err
         );
     }
+
+    use std::env;
+    if let Ok(target) = env::var("TARGET") {
+        match target.as_ref() {
+            "asmjs-unknown-emscripten" => { },
```

Comment:
There's a syntax to avoid duplicating the => { },

Reply:
I felt the "|" syntax was inelegant. What syntax exactly do you have in mind?

Reply:
Example: https://github.com/tomaka/cpal/blob/master/src/alsa/mod.rs#L94